compiler/rustc_query_system/src/dep_graph/graph.rs

```diff
@@ -52,7 +52,7 @@ impl From<DepNodeIndex> for QueryInvocationId {
     }
 }
 
-pub(crate) struct MarkFrame<'a> {
+pub struct MarkFrame<'a> {
```

Comment:
We started getting a private-in-public error when the instrument call was removed from `try_mark_parent_green`.